platform/ext/target/nordic_nrf/common/core/faults.c

```diff
@@ -86,8 +86,10 @@ __attribute__((naked)) void SPU30_IRQHandler(void)
 #endif
 
 #ifdef NRF_MPC00
-void MPC_Handler(void)
+__attribute__((naked)) void MPC_Handler(void)
```

Comment:
Seems like this function shouldn't be `naked`: https://gcc.gnu.org/onlinedocs/gcc/ARM-Function-Attributes.html#index-naked-function-attribute_002c-ARM